pkg/fontconfig

```diff
@@ -9,6 +9,7 @@ pkgver=2
 [deps]
 freetype
 expat
+gperf
```

Comment:
that should be under deps.host, since it is required only on the host when cross-compiling